src/core/actor/EwaldGPU.cpp

```diff
@@ -28,12 +28,13 @@ void EwaldgpuForce::compute_num_k() {
         if (ix * (2 * m_num_ky + 1) * (2 * m_num_kz + 1) +
                     iy * (2 * m_num_kz + 1) + iz >=
                 0 // Half m_k-space
-            and
-            SQR(ix) * SQR(m_num_ky) * SQR(m_num_kz) +
-                    SQR(iy) * SQR(m_num_kx) * SQR(m_num_kz) +
-                    SQR(iz) * SQR(m_num_kx) * SQR(m_num_ky) <=
-                SQR(m_num_kx) * SQR(m_num_ky) *
-                    SQR(m_num_kz)) // m_k-space ellipsoid
+            and Utils::sqr(ix) * Utils::sqr(m_num_ky) * Utils::sqr(m_num_kz) +
```

Comment:
maybe here: ```using Utils::sqr```

Reply:
Agreed. Added this in the two methods with lots of `sqr`.